src/Components/Register/RegisterView.js

```diff
@@ -35,6 +39,19 @@ export function RegisterView(props) {
       >
         <RegisterContainer {...props}>
           <AlertWithContext />
+          {showNameFields && (
```

Comment:
So you don't need to do anything on the container? was already sending them if existing?

Reply:
yup, the fields logic was already there in the container